oncocnv/v7.0/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:20.04 AS download
+LABEL base_image="ubuntu:20.04" \
+      version="1" \
+      software="ONCOCNV" \
+      software.version="v7.0" \
+      about.summary="ONCOCNV - a package to detect copy number changes in Deep Sequencing data" \
+      about.home="https://github.com/BoevaLab/ONCOCNV" \
+      about.documentation="https://github.com/BoevaLab/ONCOCNV/blob/master/README.md" \
+      about.license="none" \
```

Comment:
software must have a license
no license does not mean open source nor free to share

Reply:
found it and added it to the label